web/client/src/styles/form.css

```diff
@@ -18,3 +18,11 @@
 .token-display {
   word-wrap: break-word;
 }
+
+.copy-button {
+  margin-top: 1rem !important;
+}
+
+.response-box {
+  margin-top: 1rem !important;
```

Comment:
We should not use the `!important` flag unless completely necessary. It can lead to a lot of confusion

Reply:
Fixed, but need to leave the important for the copy button.